shoop/simple_cms/admin_module/views.py

```diff
@@ -69,6 +73,14 @@ def clean(self):
 
         return data
 
+    def validate_url(self, field_name, url):
+        qs = self._get_translation_model().objects.filter(url=url)
+        if self.instance.pk:
+            qs = qs.exclude(master=self.instance)
+
+        if qs.exists():
+            self.add_error(field_name, _("URL already exists."))
```

Comment:
`self.add_error(field_name, ValidationError(_(bla), code="duplicate_url"))`

You pass a `ValidationError` as the error.